docs/v3/signature_changes.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: Signature Changes
+sidebar_position: 1
+---
+
+# Signature Changes for XMTP v3
+
+## Why are signatures changing?
+
+XMTP v2 signatures use `secp256k1` keys signed by the user’s wallet. To use MLS for group chat, XMTP is migrating to `ed25519` keys signed by the `secp256k1` keys.
+
+XMTP is also requiring a signature for each app/device combination (“installation”).  Users will have to authorize XMTP for each app they install on each device (instead of once per app).
+
+## How do XMTP Identities get the new key type?
+
+When an existing user opens a V3 app for the first time, XMTP will take their `secp256k1` Create Identity signature and sign it with an `ed25519` key.
+
+New users who go through the Create Identity signature process will also get an `secp256k1` Create Identity signature signed with an `ed25519` key.
```

Comment:
@scotboyd, in order to be accurate, I think the signer/signee needs to be flipped.

```
When an existing user opens a V3 app for the first time, XMTP will create an `ed25519` key and sign it with their existing `secp256k1` key from the previously performed Create Identity step.

... etc
```